BUILD/install_build_deps.sh

```diff
@@ -153,23 +153,26 @@ sudo make install
 cd ../../
 
 # TCL
-mkdir tcl
-cd tcl
-wget -q --no-check-certificate http://prdownloads.sourceforge.net/tcl/tcl8.6.5-src.tar.gz
-
-if [ $? != 0 ]
+tcl_version=$(tclsh <<< 'puts [info patchlevel]')
+if [[ "$tcl_version" < "8.6.5" ]]
```

Comment:
Using `<` for version comparisons won't work as it does a lexicographical comparison.
```suggestion
if [[ $(echo -e "$tcl_version\n8.6.5"|sort -r -V|head -n 1) == "8.6.5" ]]
```